draft-ietf-tls-esni.md

```diff
@@ -351,7 +357,11 @@ In order to send an encrypted SNI, the client MUST first select one of
 the server ESNIKeyShareEntry values and generate an (EC)DHE share in the
 matching group. This share will then be sent to the server in the
 "encrypted_sni" extension and used to derive the SNI encryption key. It does not affect the
-(EC)DHE shared secret used in the TLS key schedule.
+(EC)DHE shared secret used in the TLS key schedule. It MUST also select
+an appropriate cipher suite from the list of suites offered by the
+server. If the client is unable to select an appropriate group or suite it SHOULD ignore that ESNIKeys value and MAY attempt to use another value provided by the server (recall that servers may provide multiple ESNIKeys in response to a ESNI TXT query).
```

Comment:
"servers may provide": lower-case "may". Change it to "might"?

Reply:
👍